.github/workflows/ci.yml

```diff
@@ -32,9 +32,9 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.10"
```

Comment:
Python 3.10 is still supported according to pyproject.toml.

Reply:
Can we drop it, or should we keep it?

Reply:
I guess we should do that in a separate PR anyways though

Reply:
I think we can drop support for Python 3.10 in a separate PR.